pom.xml

```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.alfresco</groupId>
     <artifactId>api-explorer</artifactId>
-    <version>5.2.f-SNAPSHOT</version>
+    <version>5.2.1-SNAPSHOT</version>
```

Comment:
For the January release of community (5.2.e) we decided to sync the Rest API explorer version with the alfresco platform, to avoid confusion (previous versions of Api Explorer were 1.3- plat 5.2.c , 1.4- plat 5.2.d).
We decided that is worth skipping 5.2.f(no changes) in order to keep the platform version in sync with the Api Explorer.